### DISCUSSION TITLE
Background image

### DISCUSSION THREAD

**Discussion by not-authorized** [gatsbyjs/gatsby#35686] [Ideas / Feature Requests] [answered]
Hi there! Is there any simple and properly working way to set the background image in Gatsby? Unfortunately, the GBI plugin makes multiple requests for every size of every image on a page. Spent a whole day finding the solution. 😭

**Comment by krismorf** [comment, marked as answer]
@not-authorized did you try this? 
https://www.gatsbyjs.com/docs/how-to/images-and-media/using-gatsby-plugin-image/#background-images

**Reply by not-authorized** [reply]
Thanks! 👍
